crates/block-producer/src/store/mod.rs

```diff
@@ -85,6 +85,8 @@ pub struct TransactionInputs {
     pub nullifiers: BTreeMap<Nullifier, Option<NonZeroU32>>,
     /// List of unauthenticated notes that were not found in the store
     pub missing_unauthenticated_notes: Vec<NoteId>,
+    /// The current block height
+    pub current_block_height: Option<u32>,
```

Comment:
Should this be optional?

Reply:
Good point! Seems like this should always be a `u32`.